integreat_cms/static/src/js/pois/opening-hours/component/opening-hours-display.tsx

```diff
@@ -35,6 +35,7 @@ const OpeningHoursDisplay = ({ openingHours, setSelectedDays, translations, days
                     <div class="text-right">
                         {dayOpeningHours.closed && translations.closedLabel}
                         {dayOpeningHours.allDay && translations.allDayLabel}
+                        {dayOpeningHours.appointmentOnly && translations.appointmentOnlyLabel}
```

Comment:
Maybe we need a new line in case closed or open all day is checked as well?

![Screenshot 2023-10-12 at 11-40-02 Integreat Redaktionssystem](https://github.com/digitalfabrik/integreat-cms/assets/16021269/36459977-21e0-411c-8199-128f9afcc5e5)

Reply:
My solution avoids unwanted line breaks at the cost of some empty `span`s. Please lmk if that's OK.

Reply:
Yes, looks good! :+1: